DVSRegister/Views/PreRegistration/ApplicationComplete.cshtml

```diff
@@ -4,7 +4,36 @@
     Layout = "~/Views/Shared/_Layout.cshtml";
 }
 
-<h1>ApplicationComplete</h1>
-
-
-
+<div class="govuk-width-container">
+    <main class="govuk-main-wrapper govuk-main-wrapper--l" id="main-content" role="main">
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">
+                <div class="govuk-panel govuk-panel--confirmation">
+                    <h1 class="govuk-panel__title">
+                        Application complete
+                    </h1>
+                </div>
+                <p class="govuk-body">
+                    Your information has been submitted to the <a href="{URL}" class="govuk-link">Office of Digital Identities and Attributes (OfDIA)</a>.
+                </p>
+                <p class="govuk-body">
+                    We have sent a confirmation of your application to sponsorname@email.com and username@email.com.
```

Comment:
@qumberhussain sponsor email and contact email to be populated from model. Include @model for sponsorviewmodel. if sponsor email not null or empty, show both sponsor email and contact email else show only contact email. Please take a pull/ rebase from develop before this change